<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelLinePointer</class>
 <widget class="QWidget" name="PanelLinePointer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>302</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>274</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>274</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="chkAntiAlias">
     <property name="toolTip">
      <string>Using this option will introduce weights in the selected data.</string>
     </property>
     <property name="text">
      <string>Antialiacing</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinRayNum">
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="toolTip">
      <string>More rays will be used for the tracing of the tranverse plane. This will introduce more lateral pixels. More the process of selection will be slower.</string>
     </property>
     <property name="text">
      <string>Num of Rays:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="toolTip">
      <string>This set of options adjusts how close the selected points will be to STIR coordicated system.</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioNormFid">
        <property name="toolTip">
         <string>The selected Pixels reflect any location within the pixel's boundaries.</string>
        </property>
        <property name="text">
         <string>Nor&amp;mal fidelity</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="radioHighFid">
        <property name="toolTip">
         <string>The selected locations are exactly the centers of the selected indices.</string>
        </property>
        <property name="text">
         <string>&amp;High fidelity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
